conf/messages.cy

```diff
@@ -581,11 +581,13 @@ settlorsUKAddress.checkYourAnswersLabel = Beth yw cyfeiriad hysbys diwethaf {0}?
 settlorsUKAddress.heading = Beth yw cyfeiriad hysbys diwethaf {0}?
 settlorsUKAddress.title = Beth yw cyfeiriad hysbys diwethaf y setlwr?
 
-setUpAfterSettlorDiedYesNo.checkYourAnswersLabel = A gafodd yr ymddiriedolaeth ei chreu ar ôl i’r setlwr farw?
-setUpAfterSettlorDiedYesNo.error.required = Dewiswch ‘Iawn’ os cafodd yr ymddiriedolaeth ei chreu ar ôl i’r setlwr farw
-setUpAfterSettlorDiedYesNo.heading = A gafodd yr ymddiriedolaeth ei chreu ar ôl i’r setlwr farw?
-setUpAfterSettlorDiedYesNo.hint = Gelwir hyn hefyd yn ymddiriedolaeth ewyllys.
-setUpAfterSettlorDiedYesNo.title = A gafodd yr ymddiriedolaeth ei chreu ar ôl i’r setlwr farw?
+setUpByLivingSettlorYesNo.title = TBC
```

Comment:
Need to update these messages when the Welsh descriptions have been updated [here](https://docs.google.com/spreadsheets/d/1Pf_5_p6T9MhQgoc9f9ocZyCxdoahBYLO3mQJXyvi_So/edit#gid=0).